webapp/components/_new/generic/UserAvatar/UserAvatar.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div :class="['user-avatar', size && `--${this.size}`]">
+    <span class="initials">{{ userInitials }}</span>
+    <base-icon v-if="isAnonymous" name="eye-slash" />
+    <img
+      v-else
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T15:56:50Z" title="Wednesday, January 8th 2020, 4:56:50 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,106 @@
+<template>
+  <div :class="[`size-${this.size}`, 'user-avatar']">
+    <span v-if="!hasImage || error" class="no-image">
+      <span class="flex-item">
+        <template v-if="isAnonymous">
+          <base-icon name="eye-slash" />
```


This is deeply nested. I'd try to get the icon to the same level as the image!